core/frontend/src/ViewState.ts

```diff
@@ -1642,33 +1661,31 @@ export abstract class ViewState3d extends ViewState {
     return this.getEyePoint().plusScaled(this.getZVector(), -1.0 * this.getFocusDistance(), result);
   }
 
-  /** Position the camera for this view and point it at a new target point.
-   * @param eyePoint The new location of the camera.
-   * @param targetPoint The new location to which the camera should point. This becomes the center of the view on the focus plane.
-   * @param upVector A vector that orients the camera's "up" (view y). This vector must not be parallel to the vector from eye to target.
-   * @param newExtents  The new size (width and height) of the view rectangle. The view rectangle is on the focus plane centered on the targetPoint.
-   * If newExtents is undefined, the existing size is unchanged.
-   * @param frontDistance The distance from the eyePoint to the front plane. If undefined, the existing front distance is used.
-   * @param backDistance The distance from the eyePoint to the back plane. If undefined, the existing back distance is used.
-   * @param opts for providing onExtentsError
-   * @returns A [[ViewStatus]] indicating whether the camera was successfully positioned.
-   * @note If the aspect ratio of viewDelta does not match the aspect ratio of a Viewport into which this view is displayed, it will be
-   * adjusted when the [[Viewport]] is synchronized from this view.
-   */
-  public lookAt(eyePoint: XYAndZ, targetPoint: XYAndZ, upVector: Vector3d, newExtents?: XAndY, frontDistance?: number, backDistance?: number, opts?: ViewChangeOptions): ViewStatus {
-    return this.lookAtPerspectiveOrOrtho({ eyePoint, targetPoint, upVector, newExtents, frontDistance, backDistance, opts });
-  }
-
-  /** Setup view state for either perspective or othographic view.
-   * @param args see [[LookAtPerspectiveArgs]] and [[LookAtOrthoArgs]].
+  /** Setup view state for either perspective or orthographic view.
    * @returns A [[ViewStatus]] indicating whether the camera was successfully positioned.
    * @note If the aspect ratio of viewDelta does not match the aspect ratio of a Viewport into which this view is displayed, it will be

```

Comment:
@MarcNeely for LookAtPerspective we *do* allow aspect ratio changes. As this note says, when you use a ViewState3d in a view we adjust the aspect ratio to match the screen.

Reply:
@kabentley for reference: the [BCF definition of an orthographic camera](https://github.com/buildingSMART/BCF-XML/tree/master/Documentation#orthogonalcamera) that @JoeZman is required to support. LookAtOrthoArgs doesn't have to match these, but it has to be possible for Joe to create a LookAtOrthoArgs from these parameters.
@MarcNeely BCF ortho parameters include aspectRatio but you appear to use the view's existing aspect ratio instead - why is that?

Reply:
Hmmm....  I guess I didn't realize that the view state wasn't really tied to the screen size.  In that case we could have them pass in either the verticalSize and the aspect (and leave off the extents), or just send in the extents.  But sending in the verticalSize and the extents is a little redundant since the vertical size is the y extent, so if you pass in both you have to choose which one to ignore.  Kinda the same with passing in the aspect and both extents.  One would have to take precedence over the other.

Reply:
That's why I'm suggesting for ortho, take neither aspectRatio nor verticalSize. Instead, require extents. @JoeZman can compute extents from BCF as `[aspectRatio*viewToWorldScale, viewToWorldScale]`.

Reply:
see latest. `newExtents` is optional for all permutations.

Reply:
Looks good to me - @MarcNeely please resolve this comment if you agree.